### PR TITLE
Update streams-bootstrap autoscaling config

### PR DESCRIPTION
**Pull request by disrupted**
Related to #74

### CODE REVIEW

File: kpops/components/streams_bootstrap/streams/model.py [hunk 1]
```diff
@@ -52,10 +53,23 @@ def dict(
...
+        description="Name of the consumer group used for checking the offset on the topic and processing the related lag.",
+    )
+    lag_threshold: int = Field(
+        title="Lag threshold",
+        description="Average target value to trigger scaling actions.",
+    )
+    topics: list[str] = Field(
+        default=[],
+        description="List of auto-generated Kafka Streams topics used by the streams app.",
+    )
```
Comment (disrupted) [inline kpops/components/streams_bootstrap/streams/model.py:70]: should we add the other attributes as well?
https://github.com/bakdata/streams-bootstrap/tree/master/charts/streams-app

Comment (philipp94831) [reply kpops/components/streams_bootstrap/streams/model.py:70]: Sounds good. I don't think anything will break. Only down-side is that KPOps could become harder to use with older/newer chart versions if we are too strict

File: kpops/components/streams_bootstrap/streams/model.py [hunk 2]
```diff
@@ -52,10 +53,68 @@ def dict(
...
+    offset_reset_policy: str = Field(
+        default="earliest",
+        title="Offset reset policy",
+        description="The offset reset policy for the consumer if the the consumer group is not yet subscribed to a partition.",
+    )
+    min_replicas: int = Field(
+        default=0,
+        title="Min replica count",
+        description=(
+            "Minimum number of replicas KEDA will scale the resource down to. "
```
Comment (yannick-roeder) [inline kpops/components/streams_bootstrap/streams/model.py:92]:
```suggestion
            "Minimum number of replicas KEDA will scale the resource down to."
```

File: kpops/components/streams_bootstrap/streams/model.py [hunk 3]
```diff
@@ -52,10 +53,68 @@ def dict(
...
+        description=(
+            "This is the interval to check each trigger on. "
+            "https://keda.sh/docs/2.7/concepts/scaling-deployments/#pollinginterval"
+        ),
+    )
+    cooldown_period: int = Field(
+        default=300,
+        title="Cooldown period",
+        description=(
+            "The period to wait after the last trigger reported active before scaling the resource back to 0. "
```
[the remark this answers is not in the document]
Comment (disrupted) [reply kpops/components/streams_bootstrap/streams/model.py:79]: that's on purpose for the concatenation in this multi-line string. From those descriptions the pipeline.yaml schema is generated

**Review by sujuka99 — APPROVED** [review]
LGTM, but pls wait for one more

**Review by raminqaf — APPROVED** [review]
LGTM

**Review by yannick-roeder — APPROVED** [review]

**Pull request merged**